packages/zero-runtime/README.md

```diff
@@ -269,7 +269,7 @@ Theming is an **optional** feature that lets you reuse the same values, such as
 
 > **💡 Good to know**:
 >
-> The **theme** object are used at build time without relying on React context like common CSS-in-JS libraries. This means that components created by zero-runtime `styled` will be React Server Component by default and still get benefits from theming.
+> The **theme** object is used at build time and does not exist in the final JS bundle. This means that components created using zero-runtime `styled` can be using with React Server Components by default while still getting the benefits of theming.
```

Comment:
```suggestion
> The **theme** object is used at build time and does not exist in the final JS bundle. This means that components created using zero-runtime's `styled` can be used with React Server Components by default while still getting the benefits of theming.
```

I think we need to make it clear that the `styled` function we're referring to consists of something _from_ the zero-runtime package!